src/common/config_parser.cpp

```diff
@@ -330,7 +330,17 @@ void ConfigParser::addOptionsModel(cli::CLIWrapper& cli) {
 }
 
 void ConfigParser::addOptionsTraining(cli::CLIWrapper& cli) {
-  auto previous_group = cli.switchGroup("Training options");
+  auto previous_group = cli.switchGroup("Signal Handling");
+  // --sigterm is deliberately not a boolean, to allow for a consistent
+  // pattern of specifying custom signal handling in the future.
+  // (e.g., dump model but continue training upon SIGUSR1, or report current
+  // training status upon SIGINFO.)
+  cli.add<std::string>("--sigterm",
+                       "What to do with SIGTERM: 'graceful' => save and exit (default); "
+                       "'immediate' => exit immediately.", 
+                       "graceful");
```

Comment:
Everywhere else we start help messages in a new line with four whitespaces at the beginning. Could we follow the same pattern here? Maybe you will save one line.

Reply:
Formatting fixed.